Lesson6 DZ/src/AirPassenger.java

```diff
@@ -0,0 +1,40 @@
+import java.util.Scanner;
+
+public class AirPassenger extends Air {
+    int passengers = 140;// кол-во пассажиров
+    boolean isBusiness = false; // наличие бизнес класса
+    double kilowatts;
+
+    public AirPassenger(int hp, int masSpeed, int mass, String brand, int wingspan, int minStriplength, int passengers, boolean isBusiness) {
+        super(hp, masSpeed, mass, brand, wingspan, minStriplength);
+        this.passengers = passengers;
+        this.isBusiness = isBusiness;
+    }
+
+    void hpChange() {
```

Comment:
this method is the same everywhere, so I think you can put it in the super class and make protected